[2024-03-11 04:41:17,424] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-11T04:41:13.125619+00:00 [queued]>
[2024-03-11 04:41:17,439] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-11T04:41:13.125619+00:00 [queued]>
[2024-03-11 04:41:17,439] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-11 04:41:17,439] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-11 04:41:17,439] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-11 04:41:17,448] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-11T04:41:13.125619+00:00
[2024-03-11 04:41:17,453] {standard_task_runner.py:52} INFO - Started process 1183 to run task
[2024-03-11 04:41:17,456] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting_project492', 'merge_data', '2024-03-11T04:41:13.125619+00:00', '--job-id', '659', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp371g948l', '--error-file', '/tmp/tmprr4aqrke']
[2024-03-11 04:41:17,458] {standard_task_runner.py:77} INFO - Job 659: Subtask merge_data
[2024-03-11 04:41:17,496] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting_project492.merge_data 2024-03-11T04:41:13.125619+00:00 [running]> on host a354bf513529
[2024-03-11 04:41:17,543] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting_project492
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-11T04:41:13.125619+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-11T04:41:13.125619+00:00
[2024-03-11 04:41:17,578] {python.py:151} INFO - Done. Returned value was: None
[2024-03-11 04:41:17,587] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting_project492, task_id=merge_data, execution_date=20240311T044113, start_date=20240311T044117, end_date=20240311T044117
[2024-03-11 04:41:17,629] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-11 04:41:17,649] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
